feat: allow to configure product.json using Config Map (#2877)
codenvy/che-docs: modules/administration-guide/pages/editor-configurations-for-microsoft-visual-studio-code.adoc

* settings.json
* extensions.json
* product.json

> The *settings.json* section contains various settings that allow to customize different parts of the Code - OSS editor. +

Reviewer: ```suggestion
The *settings.json* section contains various settings with which you can customize different parts of the Code - OSS editor. +
```

> The *extensions.json* section contains recommended extensions are installed when a workspace is started. +

Reviewer: ```suggestion
The *extensions.json* section contains recommended extensions that are installed when a workspace is started. +
```
